tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -141,14 +141,32 @@ Given('the following files have been deleted by user {string}', async function (
         webdav.delete(user, entry)
     )
   )
+  return client.page.filesPage()
+})
+
+Then('it should not be possible to delete file {string} using the webUI', function (resource) {
+  return client.page.FilesPageElement.filesList().isActionDisabled(resource, 'delete', (isDisabled) => {
+    client.assert.strictEqual(isDisabled, true, `Delete action must not be enabled for "${resource}"`)
+  })
+})
 
+Given('the following files have been deleted by user {string}', function (user, table) {
+  for (const line of table.rows()) {
+    webdav.delete(user, line[0])
```

Comment:
Use await to make sure files have been finished deleting before the next step starts

Reply:
obsolete